integration-tests/maven/src/test/resources/projects/test-module-dependency/app/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
```

Comment:
Do we really need this? I think we likely have a heap of these in the tests that are not actually used.

Reply:
In this specific test we don't. A typical devmode test in our testsuite though would first check the static html response. And then the rest.

Reply:
There is plenty of resources we could cleanup. Like docker files, native tests, pom's including native profiles that aren't used in our tests.